### PR TITLE
[pipelines] Added support for customVolumes, customVolumeMounts, configMaps and customInitcontainers for Vault

### PR DESCRIPTION
**Pull request by vivekkumar-git**
#### PR Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] Chart Version bumped
- [x] CHANGELOG.md updated
- [x] Variables and other changes are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[artifactory]`)

**What this PR does / why we need it**:

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

### CODE REVIEW

File: stable/pipelines/CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,10 @@
 # JFrog Pipelines Chart Changelog
 All changes to this chart to be documented in this file.
 
+## [1.3.9] - July 31, 2020
+* Added support for customVolumes, configMaps and customInitcontainers for Vault
```
Comment (danielezer) [inline stable/pipelines/CHANGELOG.md:5]: What about customVolumeMounts?

Comment (chukka) [reply stable/pipelines/CHANGELOG.md:5]: @danielezer These are already done for `pipelines` may not needed for `vault`

Comment (danielezer) [reply stable/pipelines/CHANGELOG.md:5]: What would you use custom volumes for if you don’t have custom volume mounts

Comment (chukka) [reply stable/pipelines/CHANGELOG.md:5]: Added

**Pull request merged**